unstract/connectors/src/unstract/connectors/databases/bigquery/bigquery.py

```diff
@@ -127,3 +178,27 @@ def execute_query(
                 schema=schema,
                 table_name=table,
             ) from e
+
+    def get_information_schema(self, table_name: str) -> dict[str, str]:
+        """Function to generate information schema of the big query table.
+
+        Args:
+            table_name (str): _description_
+
+        Returns:
+            dict[str, str]: _description_
+        """
+        bigquery_table_name = str.lower(table_name).split(".")
+        database = bigquery_table_name[0]
```

Comment:
Can you also include the expected format of `table_name` in the docstring? It would help clarify how the `table_name` should be structured (e.g., `"database.schema.table"`) for this code to work correctly.